### ISSUE TITLE
Ensure modals are a11y awesome

### ISSUE THREAD

**Issue by DHedgecock** [crystal-ball/componentry#106]
Take a look at this modal specifically built to be a11y a++ and see if we're missing any requirements.

https://micromodal.now.sh/
